lib/resources/kernel_module.rb

```diff
@@ -37,13 +66,56 @@ def loaded?
       !found.nil?
     end
 
+    # @note maintain logcal agreement with 'disabled?' method
+    alias enabled? loaded?
+
+    def blacklisted?
+      if inspec.os.redhat? || inspec.os.name == 'fedora'
+        modprobe_cmd = "/sbin/modprobe --showconfig | grep blacklist | grep #{@module}"
+      else
+        modprobe_cmd = "modprobe --showconfig | grep blacklist | grep #{@module}"
+      end
+      cmd = inspec.command(modprobe_cmd)
+      cmd.exit_status.zero? ? cmd.stdout.delete("\n") : nil
```

Comment:
So, is this a better approach?
```
cmd = inspec.command(modprobe_cmd)
cmd.exit_status.zero? ? true: nil
```
or, to be fully within the expected behaviour of the `?` convention.
```
cmd = inspec.command(modprobe_cmd)
cmd.exit_status.zero? ? true: false
```
i.e. if the command completed successfully, then return true, else return nil.